vuepress/docs/next/tutorials/cms/digital-assets-tutorial.md

```diff
@@ -1,30 +1,23 @@
 # Digital Assets
 
-## Objective
-
-This document provides an overview of the Digital Assets and
-functionalities needed to manage images and documents.
-
 ## Overview
 
-The WEB CMS includes capabilities that let you manage contents and
-digital assets. Digital Assets are images, documents, and other media
-files supported in specific format.
+The Web Content Management System (WCMS) includes capabilities that let you manage contents and digital assets. Digital Assets are images, documents, and other media
+files in specific formats.
 
-**Default Supported Document Formats:** The Entando CMS can be
+**Default Supported Document Formats:** The Entando WCMS can be
 configured to allow you to load or exclude any type of file extension
-from being loaded to your site. For the app builder you should also include the MIME type of the asset you want to allow. The default set of extensions are:
-
-pdf,xls,doc,ppt,txt,rtf,sxw,sxc,odt,ods,odp,tar,gz,zip,rar,flv,swf,avi,wmv,ogg,mp3,wav,ogm,mov,iso,nrg,docx,docm,xlsx,xlsm,xlsb,pptx,pptm,ppsx,ppsm,sldx,sldm.
+from being loaded to your site. For the app builder you should also include the MIME type of the asset you want to allow. The default set of extensions are: 
+`pdf,xls,doc,ppt,txt,rtf,sxw,sxc,odt,ods,odp,tar,gz,zip,rar,flv,swf,avi,wmv,ogg,`
+`mp3,wav,ogm,mov,iso,nrg,docx,docm,xlsx,xlsm,xlsb,pptx,pptm,ppsx,ppsm,sldx,sldm`
 
-**Default Supported Image Formats**: jpg, jpeg, png, svg,svg+xml
+**Default Supported Image Formats**: `jpg, jpeg, png, svg,svg+xml`
 
-## How to create a Digital Asset
+## Create a Digital Asset
 
-To manage the Digital Assets through the Web CMS integrated with App
-Builder application
+To manage the Digital Assets through the WCMS:
 
-- From App Builder, go to `Content > Types`
+- Go to `Content > Types`
```

Comment:
A very very small thing but we should standardize how we're representing that menu transition. I've been doing "->" this one has ">", I've seen "-", and I've also done "-->".

Not something to fix now but a note

Reply:
Good call, it's all over the map. I'm fine to standardize on -> if you prefer it and can keep an eye open for it as I'm updating docs.